rusk/src/lib/http/event.rs

```diff
@@ -0,0 +1,228 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use serde::{Deserialize, Serialize};
+use serde_with;
+use serde_with::serde_as;
+use std::fmt::{Display, Formatter};
+
+/// A request sent by the websocket client.
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct WsRequest {
```

Comment:
I think this structure should be called just `Request`, since it's also used in HTTP

```suggestion
pub(crate) struct Request {
```